runtimes/pythonrt/py-sdk/autokitteh/atlassian.py

```diff
@@ -15,7 +14,7 @@
 __TOKEN_URL = "https://auth.atlassian.com/oauth/token"
 
 
-def atlassian_jira_client(connection: str, **kwargs) -> Jira:
+def jira_client(connection: str, **kwargs) -> Jira:
```

Comment:
This is an API breaking change. 
Do we know if there's a current client who's using it?
If so, keep the old function as well, something like:

```python
import warnings

def atlassian_jira_client(connection: str, **kwargs) -> Jira:
    warnings.warn("`attlasian_jira_client` is deprecated, use `jira_client`", DeprecationWarning)
    return jira_client(connection, **kwargs)

```

Reply:
@daabr I don't think anyone but us is using this correct?

Reply:
We have one customer who requested the Jira integration, but they don't have any projects that use it yet. The only places that reference this are our projects in the `samples` and `kittehub` repos - and there are PRs to fix them, which will be merged as soon as this PR is merged.

Reply:
Go ahead then.